machina/templates/machina/forum_member/forum_profile_update.html

```diff
@@ -32,6 +32,18 @@ <h3 class="panel-title">{% trans "Edit forum profile" %}</h3>
                         </div>
                         <div class="row">
                             <div class="col-md-12">
+                                {% include "partials/form_field.html" with field=form.auto_subscribe_topics %}
+                            </div>
+                        </div>
+                        {% if MACHINA_ENABLE_EMAIL_NOTIFICATIONS %}
```

Comment:
IMHO this is not the right way to do it. Users could still update the value of the notification-related booleans if they want to. Fields required to manage notifications should be dynamically inserted in the form instance. Thus there'd be no need for the presence of the `MACHINA_ENABLE_EMAIL_NOTIFICATIONS` in the machin's main context processor.